### ISSUE TITLE
Clearer explanations in the docs' getting started section

### ISSUE THREAD

**Issue by kubyon** [Vahera/godot-orchestrator#621]
### Description

At the moment, the documentation's getting started section directly mirrors that of Godot's. While this has provided a decent baseline for contents, many features/concepts unique to Orchestrator are not properly elaborated upon (e.g the All Actions dialogue box, relationships between graph nodes), making tutorials harder to follow and be understood, particularly for non-programmers. As such, the section could benefit from additional clarity.

### Implementation ideas

At the moment, it is recommended that users should review Godot's docs before proceeding to Orchestrator's. While the former has done a good job explaining concepts (especially those of technical nature), it is only relative to GDScript's features, something that Orchestrator does not yet build upon. 

As provided in the example below, the note does explain the Call Init event's functions, but does not make it clear why you should use those 2 nodes in particular or why you should be connecting white arrows to each other.
<img width="782" alt="image" src="https://github.com/user-attachments/assets/8e07fd8d-fbf4-4e07-b8d2-e177850f7a14">
Personally, I think _key Orchestrator features could be briefly explained right after being introduced_, to make it clearer why they function as such.

The icon rotation tutorial faces a similar problem. It tells you what steps you should take to get this result:
![image](https://github.com/user-attachments/assets/b942bf09-040f-4ae3-bf21-abe0398945e0)
...But it does not explain why you should use those nodes, nor why are nodes connected as such.
_An explanation of node relationships could make concepts easier to digest as well._ I was only able to understand this after cross-referencing both docs multiple times, which is not very ideal for beginners.

**Comment by Naros** [comment]
Hi @kubyon, would you be willing to submit a PR to the https://github.com/Vahera/godot-orchestrator-docs repository, helping to improve the documentation based on the points you outlined above?

**Comment by kubyon** [comment]
@Naros Ideally, I would love to make the changes myself. However, my limited programming experience means I could misunderstand important concepts and make misleading additions. Given that this plugin is in active development, would this be a major concern?

**Comment by Naros** [comment]
Not at all.  In fact, all changes are reviewed by maintainers before being committed, so if something needs to be changed, one of us will let you know in the pull request review, open collaboration at its best! 👍

**Comment by kubyon** [comment]
I'll see what I can do during this month then - it seems like a better idea to get my feet wet in GDScript first before proceeding. Thanks for the heads up!
